### PR TITLE
DOC-2063: Added TINY-9633 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2063

Changes:
* Added TINY-9633 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -73,6 +73,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
+//#TINY-9633
+
+In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab on the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
+
+In {productname} 6.5, a new pipeline has been implemented for delivering translations for both long strings and professional translations for text within the “Keyboard Navigation” tab inside the Help dialog.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.5-release-notes.adoc:81]:
```suggestion
In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the “Keyboard Navigation” tab inside the Help dialog have also been procured.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -73,6 +73,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
+
+As a result, the text is now shown in the language specified by the language option.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.5-release-notes.adoc:83]:
```suggestion
As a result, the text in the Help dialog's "Keyboard Navigation" tab is now shown in the language specified by the `language` option.
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -73,6 +73,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
+//#TINY-9633
+
+In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab on the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.5-release-notes.adoc:79]:
```suggestion
In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
```

**Review by mkzhx — APPROVED** [review]
Looks nice, the note at the end is a good addition 👍

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 4]
```diff
@@ -73,6 +73,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
+//#TINY-9633
+
+In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
+
+In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the “Keyboard Navigation” tab inside the Help dialog have also been procured.
+
+As a result, the text in the Help dialog's "Keyboard Navigation" tab is now shown in the language specified by the `language` option.
+
+[NOTE]
+Due to the large differences between this new pipeline and conventional methods of delivering translations, the professional translations for the text on the “Keyboard Navigation” tab have been made **open-source** in the `i18n` subdirectory in the Help plugin for _community users_.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:86]:
```suggestion
=== Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
//#TINY-9633

In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered.

Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the *Keyboard Navigation* tab.

In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the *Keyboard Navigation* tab inside the **Help** dialog have also been procured.

As a result, the text in the **Help** dialog’s *Keyboard Navigation* tab is now shown in the language specified by the current editor’s `language` option.

NOTE: Due to the large differences between this new pipeline and conventional methods of delivering translations, the professional translations for the text on the *Keyboard Navigation* tab have been made open source for community users. These translations are in the `i18n` subdirectory of the **Help** plugin.
```

Copy-edits.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**